Essentials/src/plugin.yml

```diff
@@ -571,9 +571,6 @@ permissions:
     description: Give players with op everything by default
     children:
       essentials.gamemode.*: true
-  essentials.sleepingignored:
```

Comment:
JRoy apparently removed this in #3337, he removed the other one however

Reply:
yo, can we merge #3337

Reply:
Hey guys, did something go wrong here? Both entries still seem to be removed to this day.